packages/kotti-ui/source/kotti-navbar/KtNavbar.vue

```diff
@@ -146,17 +149,35 @@ $narrow-navbar-width: 3.4rem;
 		--user-menu-color: var(--primary-80);
 	}
 
-	&--theme-grayscale {
+	&--theme-light {
```

Comment:
in general, i have a question

do we customize the primary colors to our customers
so if we want to extend themes to fit our customers requirements, do we only need to change what primary-* values are

Reply:
yes we do